### PR TITLE
Web Health Study consent (bug 1378555) fixes #5875

### LINKED ISSUE #5875: Shield Study 13 - Web Health Study (bug 1378555)
We would like to host a consent page for the Web Health Shield Study (per https://bugzilla.mozilla.org/show_bug.cgi?id=1378555).

### PR DESCRIPTION
**Pull request by raymak**
fixes #5875

Tracking bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1378555

### CODE REVIEW

File: src/olympia/pages/templates/pages/shield_study_13.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>Understanding the Web</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_13/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Understanding the Web</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/832183/addon-832183-latest.xpi"><button class="button primary large">Start Now</button></a>
```
Comment (EnTeQuAk) [inline src/olympia/pages/templates/pages/shield_study_13.html]: This links to an unsigned add-on apparently? At least I'm unable to install it in nightly, is this expected?

File: src/olympia/pages/templates/pages/shield_study_13.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+<h2 id="this-particular-study">This Particular Study</h2>
+<p>In addition to the data collected by all Shield Studies, here are the key things you should know about what is happening when you participate in this study:</p>
+<ul>
+<li>We will record the number of times you visit certain pre-defined set of websites (only top-level hostnames will be collected, e.g. www.amazon.com)</li>
+<li>We will record information about your interactions with notifications coming from this study</li>
+<li>The study will not be active, and no information will be recorded, while using Private Browsing Mode</li>
+<li>We will not collect any identifying information such as names, phone numbers, or email addresses</li>
+<li>This study is covered under IRB #827604 from the University of Pennsylvania</li>
+</ul>
+<p>You can learn more about the data collection for this study, including the pre-defined list of websites, <a href="https://github.com/raymak/shield-internet-policy-awareness/wiki/Data-Collection">here</a>.</p>
```
Comment (EnTeQuAk) [inline src/olympia/pages/templates/pages/shield_study_13.html]: `here` as a link text is very bad for accessibility reasons, the link text should describe the target. Can we change that to something more useful, e.g `in our Wiki` or something like that?

**Comment by raymak** [comment]
The addon link is expected. The addon is awaiting review but that will be
there link once it's ready. Regarding the wording, I will hopefully fix it
by the morning. If I didn't, please go ahead with the merge as it's
critical to launch the study this week. Thanks!

**Comment by raymak** [comment]
The new commit should fix the wording issue.

**Comment by EnTeQuAk** [comment]
squashed and merged, thanks for the update!

**Pull request merged**